### ISSUE TITLE
Question to the Project

### ISSUE THREAD

**Issue by MorrisBrProjects** [daniel-dona/beken7252-opencam#1]
Help the project with my Little Star app camera?
Id start with ACCQ

Maybe this, I'm not sure, but it looks the same
https://www.amazon.de/SALUCIA-WiFi-%C3%9Cberwachungskamera-Intelligente-Nachtsicht-Bewegungserkennung-Au%C3%9Fenbereich/dp/B0B3WZ85YC

**Comment by MorrisBrProjects** [comment]
I have try to decode and send self and get viedio Data from Camera, but i failed.

But how install custom firmware on the chip?

**Comment by MorrisBrProjects** [comment]
@daniel-dona ?

**Comment by Bluscream** [comment]
> The only important thing is that the SoC is a BK7252, board revisions or variants are OK, but it looks like not all the A9 cameras use the BK7252.
> 
> In fact, I just got another one that has a TXW806-840 chip... https://www-taixin--semi-com.translate.goog/Product/ProductDetail?productId=285&_x_tr_sl=auto&_x_tr_tl=es&_x_tr_hl=es&_x_tr_pto=wapp
> 
> There are at least two different ways of identifying the SoC vendor:
> 
> 1. Using the MAC address and a MAC database: https://macvendors.com/, if it says Beken, most probably is a BK7252
> 2. Just opening the product. In most cases, you only need to remove two small screws and take a look at the letters on the SoC, should look something like this:
>    ![imagen](https://user-images.githubusercontent.com/23705091/270662470-b0630e40-654b-4ded-ad70-2eb463683dd9.png)

what about this one? will it work for this?

![](https://community-assets.home-assistant.io/original/4X/a/1/a/a1a4a5fb87ef45b9ca9ae58a71f641d55a8f1128.jpeg)
https://community.home-assistant.io/t/popular-a9-mini-wi-fi-camera-the-ha-challenge/230108/248?u=bluscream

**Comment by daniel-dona** [comment]
> I have try to decode and send self and get viedio Data from Camera, but i failed.
> 
> But how install custom firmware on the chip? How can i access and override firmware?
> 
> And i have 1:1 the same chip how in the picture
> 
> But I don't have to desolder and resolder the chip, do I?

I'm working on documenting flashing procedure, PCB variations, etc... but keep in mind that the current code is only a base from where to build a functional firmware, is not a working alternative already...

**Comment by daniel-dona** [comment]
> > 2. Just opening the product. In most cases, you only need to remove two small screws and take a look at the letters on the SoC, should look something like this:
> 
> what about this one? will it work for this?
> 
> ![](https://camo.githubusercontent.com/2cf4c0a2cc9f2698acea67543896eb746a568ec06fd4667ef9a8fb7999cd9fbc/68747470733a2f2f636f6d6d756e6974792d6173736574732e686f6d652d617373697374616e742e696f2f6f726967696e616c2f34582f612f312f612f613161346135666238376566343562396361396165353861373166363431643535613866313132382e6a706567) https://community.home-assistant.io/t/popular-a9-mini-wi-fi-camera-the-ha-challenge/230108/248?u=bluscream

Looks like a Beken chip, so yes, but a detailed pinout whould be required to use the other chips, that look like motor drivers

**Comment by MorrisBrProjects** [comment]
> > I have try to decode and send self and get viedio Data from Camera, but i failed.
> > But how install custom firmware on the chip? How can i access and override firmware?
> > And i have 1:1 the same chip how in the picture
> > But I don't have to desolder and resolder the chip, do I?
> 
> I'm working on documenting flashing procedure, PCB variations, etc... but keep in mind that the current code is only a base from where to build a functional firmware, is not a working alternative already...

hm, can you make a version finish to use?
im no a c, c++ developer.

**Comment by Bluscream** [comment]
> > > 2. Just opening the product. In most cases, you only need to remove two small screws and take a look at the letters on the SoC, should look something like this:
> > 
> > 
> > what about this one? will it work for this?
> > ![](https://camo.githubusercontent.com/2cf4c0a2cc9f2698acea67543896eb746a568ec06fd4667ef9a8fb7999cd9fbc/68747470733a2f2f636f6d6d756e6974792d6173736574732e686f6d652d617373697374616e742e696f2f6f726967696e616c2f34582f612f312f612f613161346135666238376566343562396361396165353861373166363431643535613866313132382e6a706567) https://community.home-assistant.io/t/popular-a9-mini-wi-fi-camera-the-ha-challenge/230108/248?u=bluscream
> 
> Looks like a Beken chip, so yes, but a detailed pinout whould be required to use the other chips, that look like motor drivers

Yeah it has IR leds and tilt/pan motors. Thats mainly why i bought it
